tasty/graphs.py

```diff
@@ -144,3 +146,81 @@ def get_namespaced_term(ontology: Graph, term: str) -> Union[URIRef, bool]:
         ns = potential_namespaces[0]
         return ns[term]
     return False
+
+
+def print_as_hayson(graph: Graph):
+    """
+    Return the Hayson encoding of an rdf graph
+    :param graph: [rdflib.Graph]
+    :return:
+    """
+    hayson = {
+
+        "meta": {"ver": "3.0"},
+        "cols": [],
+        "rows": []
+    }
+    json_ld = graph.serialize(format='json-ld').decode('utf-8')
+    data = json.loads(json_ld)
+    for row in data:
+        hayThing = {"id": str(uuid.uuid4())}
+        for key, value in row.items():
+            if key == "@id":
+                dis = value
+                hayThing.update({"dis": dis})
+            elif key == "@type":
+                tags = value[0].split('#')[1]
+                multitag = tags.split("-")
+                for tag in multitag:
+                    hayThing.update({tag: "m"})
```

Comment:
Could this conditional be done with a single line list comprehension?

Reply:
maybe `hay_thing.update({tag: "m"}) for tag in multitag`? Or something close to that?

Reply:
Yeah, that's what I was thinking.